smartcard/pcsc/PCSCReader.py

```diff
@@ -114,6 +116,10 @@ def readers(groups=[]):
             hcontext = PCSCContext.renewContext()
             pcsc_readers = __PCSCreaders__(hcontext, groups)
 
+        except CardServiceNotFoundException:
```

Comment:
I would prefer to handle the 2 exceptions at once with:
```
except (CardServiceStoppedException, CardServiceNotFoundException):
```
to factorize the code and avoid code duplication.